spec/models/message_spec.rb

```diff
@@ -5,8 +5,8 @@
   context 'with valid attributes' do
     subject { described_class.new(params) }
 
-    let(:params) { attributes_for :message }
-    let(:other) { described_class.new(attributes_for(:message)) }
+    let(:params) { attributes_for :message_for_model }
+    let(:other) { described_class.new(attributes_for(:message_for_model)) }
```

Comment:
I'm not sure this name change is appropriate, I think this is actually more confusing. :message is standard factory naming convention, what is message_for_model? I liked what we were doing with slice better.